src/ctia/bulk/core.clj

```diff
@@ -253,6 +278,19 @@
     [:ctia :store :bulk-refresh]
     "false"))
 
+(defn bulk-size [bulk]
+  (apply + (map count (vals bulk))))
+
+(defn get-bulk-max-size [get-in-config]
+  (get-in-config [:ctia :http :bulk :max-size]))
+
+(defn bad-request?
+  [bulk
+   {{:keys [get-in-config]} :ConfigService}]
+  (when (> (bulk-size bulk) (get-bulk-max-size get-in-config))
+    (bad-request (str "Bulk max nb of entities: "
```

Comment:
Looks like this line will be included in the outgoing response. Introducing non-common shorthand might be confusing. Also how about including actual bulk size into the response body together with maximum size from the config?

```suggestion
    (bad-request (str "Bulk max number of entities: "
```